letters/letters.py

```diff
@@ -0,0 +1,67 @@
+import re
+
+from redbot.core import commands
+
+
+class Letters(commands.Cog):
+    """Letters cog"""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def letters(self, ctx, *, msg):
+        """Outputs large emote letters (\"regional indicators\") from input text.
+        Accepts A-Z, 0-9, and whitespace only.
+
+        Example:
+        - `[p]letters I'd like this text as emotes 123`
+        """
+
+        # Grab message content
+        input = msg.lower()
+
+        # Define numbers -> emotes dict
+        nums = {
+            0 : ':zero:',
+            1 : ':one:',
+            2 : ':two:',
+            3 : ':three:',
+            4 : ':four:',
+            5 : ':five:',
+            6 : ':six:',
+            7 : ':seven:',
+            8 : ':eight:',
+            9 : ':nine:'
+        }
```

Comment:
1. We can hoist this to global (line 4/5) or even as a class variable (line 8) instead of defining it every time this function is executed
2. Since we're indexing using numbers 0-9, we can actually use a list/tuple instead, which probably makes more sense.

Reply:
Resolved in 44657c2